clamav-http/server/v1alpha/scan_handler.go

```diff
@@ -0,0 +1,63 @@
+package v1alpha
+
+import (
+	"net/http"
+
+	"github.com/dutchcoders/go-clamd"
+	"github.com/sirupsen/logrus"
+)
+
+type ScanHandler struct {
+	Address      string
+	Max_file_mem int64
+	Logger       *logrus.Logger
+}
+
+func (sh *ScanHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	err := r.ParseMultipartForm(sh.Max_file_mem * 1024 * 1024)
+
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte("not okay"))
+		return
+	}
+
+	files := r.MultipartForm.File["file"]
+
+	if len(files) == 0 {
+		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
I don't think an empty file should return an error
```suggestion
		w.WriteHeader(http.StatusNoContent)
```

Reply:
Agreed, changed this